### ISSUE TITLE
Feature request: Set `noshort = true` for all options

### ISSUE THREAD

**Issue by mschwerhoff** [scallop/scallop#198]
I have a `MyConfig(...) extends ScallopConf(...)` with loads (~40) of options, none of which should have a short version. Setting `noshort = true` for each option is certainly possible, but I would be splendid if it sufficed to only set it once :-) E.g. by passing an argument to `ScallopConf` or overriding a field.

**Comment by Rogach** [comment]
I think it is possible to add `noshort` option to ScallopConf, but unfortunately I do not have the time right now.

You can get a similar effect by wrapping `.opt` call in a function:
```scala
object MyConfig(args: Seq[String]) extends ScallopConf(args) {
  def _opt[A](
    descr: String,
    default: => Option[A] = None,
    required: Boolean = false
  )(implicit conv: ValueConverter[A]) = {
    opt(
      descr = descr,
      default = default,
      required = required,
      noshort = true
    )(conv)
      
  val apples = _opt[Int]()
}
```

**Comment by mschwerhoff** [comment]
Good idea with the `_opt` wrapper, I'll give that a try. Thank you!

**Comment by Rogach** [comment]
Implemented by @zawlazaw in #206
